pwnlib/context/__init__.py

```diff
@@ -1359,11 +1359,20 @@ def LocalContext(function):
     """
     @functools.wraps(function)
     def setter(*a, **kw):
-        # Fast path to skip adding a Context frame
-        if not kw:
-            return function(*a)
-
         with context.local(**{k:kw.pop(k) for k,v in kw.items() if isinstance(getattr(ContextType, k, None), property)}):
+            arch = context.arch
+            bits = context.bits
+            end = context.endian
```

Comment:
I think you meant `endianness`
```suggestion
            endianness = context.endian
```